.github/workflows/partial-tests.yml

```diff
@@ -79,110 +78,32 @@ jobs:
 
       - name: Pytest CLI
         if: steps.changes.outputs.cli == 'true'
-        run: pipenv run pytest tests/cli
+        run: pipenv run pytest  tests/cli
 
       - name: Pytest Config
         if: steps.changes.outputs.config == 'true'
-        run: pipenv run pytest tests/config
+        run: pipenv run pytest  tests/config
 
       - name: Pytest Core
         if: steps.changes.outputs.core == 'true'
-        run: pipenv run pytest -m "not orchestrator_dispatcher and not standalone" tests/core
+        run: pipenv run pytest  tests/core
```

Comment:
I think we should keep this while our unit tests are flacky with the orcherstrator and standalome modes.

Reply:
I agree.